tt_eager/tt_dnn/op_library/moreh_linear_backward/kernels/moreh_bias_backward_multi_core_h.cpp

```diff
@@ -0,0 +1,118 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "compute_kernel_api/eltwise_binary.h"
+#include "compute_kernel_api/mask.h"
+#include "compute_kernel_api/reduce.h"
+#include "compute_kernel_api/tile_move_copy.h"
+
+ALWI void ACQ() { acquire_dst(tt::DstMode::Half); }
+ALWI void REL() { release_dst(tt::DstMode::Half); }
+
```

Comment:
We are migrating to these new APIs:
- `acquire_dst` = `tile_regs_acquire` and `tile_regs_wait`
- `release_dst` = `tile_regs_commit` and `tile_regs_release`. 

Take a look at this file for documentation: `tt_metal/include/compute_kernel_api/reg_api.h`.
And this file for examples: `tt_eager/tt_dnn/op_library/bmm/kernels/compute/bmm_large_block_zm_fused_bias_activation.cpp`

It's fine for this PR, but I think it's worth switching over.

Reply:
Thank you for your suggestion. I'll apply it to my next PR. :)